js/Board.js

```diff
@@ -0,0 +1,35 @@
+//** LED Board **//
```

Comment:
Can we do all lowercase file names?

Reply:
would you prefer lower-cased, camelCase or snake_case